source/feature/inclusterupgrade.html.md

```diff
@@ -75,10 +75,6 @@ The weight policy unit gives an OS which is newer than the OS where the VM is cu
 *   No load balancing will happen
 *   Migrating a VM back to an older OS version is not possible during upgrade. To run a VM on an older host OS, the VM hast to be stopped first.
 
-#### What might stop you from putting a host to maintenance
-
-*   Missing networks
```

Comment:
There was just an idea to collect reasons.
There are many reasons why a host can't be put to maintenance and it has nothing to do with this feature. I decided that it is counterproductive to have that here because it will just be outdated soon. Maybe we have a useful link where I can point to.